services/user-management/src/main/java/com/foodie/user/model/User.java

```diff
@@ -28,57 +24,21 @@
 @AllArgsConstructor
 @Entity
 @Table(name = "users")
-public class User extends FoodieBaseEntity implements UserDetails {
+public class User extends FoodieBaseEntity {
```

Comment:
You should implement Serializable in your model classes

Reply:
Will look into it